### PR TITLE
Fix bugs, add new method

### PR DESCRIPTION
**Pull request by dastanbeksamatov**
- Fix bug in `ScaleString.fromU8a()`
- Add `!=` operator in ByteArray
- Add unit test for updated method

### CODE REVIEW

File: assembly/__tests__/ScaleString.spec.ts [hunk 1]
```diff
@@ -41,9 +41,13 @@ describe("String", () => {
...
         expect<Array<u8>>(scaleString).toStrictEqual(([0x61]));
         expect<string>(scaleString.toString()).toStrictEqual("a");
+
+        const scaleString1 = ScaleString.fromU8a([0x08, 0x64, 0x61, 0x62, 0x62]);
+        expect<Array<u8>>(scaleString1).toStrictEqual([0x64, 0x61]);
+        expect<string>(scaleString1.toString()).toStrictEqual("da")
     });
 
-    itThrows("should throw on incorrect encoding", () => {
+    throws("should throw on incorrect encoding", () => {
```
Comment (Daniel-K-Ivanov) [inline assembly/__tests__/ScaleString.spec.ts]: why did you change it to `throws`? 
We are using `itThrows` in many places, did the syntax of the `aspect` lib changed?

Comment (dastanbeksamatov) [reply assembly/__tests__/ScaleString.spec.ts]: Don't think it matters, so I'll just change it now

**Review by Daniel-K-Ivanov — APPROVED** [review]
